### ISSUE TITLE
Should we delete from database the old (say more than 1 year old) expired invoices that didn't receive payments?

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#5832]
Should we delete from database the old (say more than 1 year old) expired invoices that didn't receive payments?

This would save a lot's of space.

_Originally posted by @NicolasDorier in https://github.com/btcpayserver/btcpayserver/discussions/5830_

**Comment by Kukks** [comment]
No, if anything this should be a plugin. I already have a plugin that
scrubs some data on invoices after some time. Can make it have an option to
flat out delete those

**Comment by pavlenex** [comment]
@NicolasDorier What do you think about adding this functionality to an existing plugin by Kukks? Seams reasonable to me as the intention of the plugin is to remove customer's data due to GDPR.
